internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -932,6 +932,22 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 
 				jsonStructData[i].(map[string]interface{})["connections"] = nil
 			}
+		case "cloudflare_turnstile_widget":
+			jsonPayload, _, err := api.ListTurnstileWidgets(context.Background(), cloudflare.AccountIdentifier(accountID), cloudflare.ListTurnstileWidgetParams{})
```

Comment:
since `ListTurnstileWidgets` automatically paginates, we don't need to do anything special here to get all the values.

Reply:
Nice!